server.go

```diff
@@ -55,64 +55,161 @@ func OptServerLogger(l Logger) ServerOption {
 	}
 }
 
+func OptServerReplicaN(n int) ServerOption {
+	return func(s *Server) error {
+		s.Cluster.ReplicaN = n
+		return nil
+	}
+}
+
+func OptServerDataDir(dir string) ServerOption {
+	return func(s *Server) error {
+		s.Cluster.Path = dir
+		s.Holder.Path = dir
+		return nil
+	}
+}
+
+func OptServerAttrStoreFunc(af func(string) AttrStore) ServerOption {
+	return func(s *Server) error {
+		s.NewAttrStore = af
+		s.Holder.NewAttrStore = af
+		return nil
+	}
+}
+
+func OptServerAntiEntropyInterval(interval time.Duration) ServerOption {
+	return func(s *Server) error {
+		s.AntiEntropyInterval = interval
```

Comment:
I didn't check all of these functional options, but we can un-export `AntiEntropyInterval` now unless there's a reason to leave it exported.